ontolearner/base/learner.py

```diff
@@ -46,6 +47,7 @@ def index(self, inputs: List[Any]):
     def retrieve(self, inputs: List[Any], top_k: int):
```

Comment:
Can we specify the output that this `retrieve` function will return? is it a List or Dict or what?